_caseStudies/2022-10-07-oss-usage-east-africa.md

```diff
@@ -0,0 +1,92 @@
+|---------------------------------|------------------------------------------------------------------------------------------------------------|-----------|
+| Data collection and management  | Open Data Kit (ODK) Collect                                                                                |     3     |
+|                                 | KoboToolbox/Kobo Collect                                                                                   |     2     |
+|                                 | SurveyCTO                                                                                                  |     2     |
+|                                 | DHIS2                                                                                                      |     1     |
+|                                 | Excel                                                                                                      |     1     |
+|                                 | MES                                                                                                        |     1     |
+| Data analysis and visualization | Excel                                                                                                      |     5     |
+|                                 | Stata                                                                                                      |     3     |
+|                                 | R                                                                                                          |     2     |
+|                                 | PowerBI                                                                                                    |     2     |
+|                                 | DHIS2                                                                                                      |     1     |
+|                                 | EPI Info                                                                                                   |     1     |
+|                                 | MES                                                                                                        |     1     |
+| Data reporting                  | No software specified, mostly channels or format of reporting such as social media, website, policy briefs |           |
+|                                 |                                                                                                            |           |
+
+
+**Challenges:**
+Respondents were asked about the challenges around using open source software, and they cited specific products, organizational issues, and M&E-specific challenges.
+
+*Products*
+- DHIS2 has gotten better in terms of data collection and visualization over the years, though it still has a ways to go. 
```

Comment:
Last part should be, though it still has _**a long**_ way to go.

Reply:
agreed